### PR TITLE
Add mime-types-data 3.2018.0812

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/tfm-ror52-rubygem-mime-types-data/tfm-ror52-rubygem-mime-types-data.spec [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/Code-of-Conduct.md
```
Comment (ekohl) [inline packages/tfm-ror52-rubygem-mime-types-data/tfm-ror52-rubygem-mime-types-data.spec]: Also a `%doc`?

**Comment by ehelms** [comment]
@ekohl Moved to %doc section

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

**Review by ekohl — APPROVED** [review]

**Pull request merged**